cccatalog-api/cccatalog/settings.py

```diff
@@ -155,11 +155,13 @@
 }
 
 # Produce CC-hosted thumbnails dynamically through a proxy.
-PROXY_THUMBS = bool(os.environ.get('PROXY_THUMBS', False))
+PROXY_THUMBS = bool(os.environ.get('PROXY_THUMBS', True))
 THUMBNAIL_PROXY_URL = os.environ.get(
-    'THUMBNAIL_PROXY_URL', 'https://localhost:8222'
+    'THUMBNAIL_PROXY_URL', 'http://thumbs:8222'
```

Comment:
Instead of changing the default environment settings (which are often used by our servers), please override them in the `environment` section of `docker-compose.yml`.

Reply:
Here's the URL that will be used by thumbnail proxy caller (i.e. Django program), so the value should be `thumbs` (except for the `thumbs` node name in `docker-compose.yml` is changed), which is the hostname to visit `thumbs` node for the `web` node.